<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HPGLPlotter</class>
 <widget class="QMainWindow" name="HPGLPlotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HPGLPlotter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1" rowspan="2">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="1">
          <widget class="QPushButton" name="button_down">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
            </font>
           </property>
           <property name="text">
            <string>↓</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="button_up">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
            </font>
           </property>
           <property name="text">
            <string>↑</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="button_right">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
            </font>
           </property>
           <property name="text">
            <string>→</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="button_topLeft">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>36</pointsize>
             <italic>false</italic>
             <bold>false</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>⇖</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="button_center">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>false</bold>
             <underline>false</underline>
             <kerning>false</kerning>
            </font>
           </property>
           <property name="text">
            <string>Center</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="button_left">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>36</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>UpArrowCursor</cursorShape>
           </property>
           <property name="text">
            <string>←</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="button_bottomRight">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
            </font>
           </property>
           <property name="text">
            <string>⇘</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="button_bottomLeft">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
            </font>
           </property>
           <property name="text">
            <string>⇙</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="button_topRight">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
            </font>
           </property>
           <property name="text">
            <string>⇗</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="pushButton_3">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>50</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Run plot with pen up</string>
           </property>
           <property name="text">
            <string>Dry Run</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="pushButton_2">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Run plot</string>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>⯈</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>32</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>Stop plot</string>
           </property>
           <property name="text">
            <string>⏹</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QTextEdit" name="text_console"/>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label_scalingFactor">
           <property name="text">
            <string>Scaling Factor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_scalingFactor">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="displayIntegerBase">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionImport_HPGL"/>
    <addaction name="separator"/>
    <addaction name="actionExport_HPGL"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="action_penConfig"/>
    <addaction name="action_plotterConfig"/>
    <addaction name="action_viewConfig"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionExport_HPGL">
   <property name="text">
    <string>Export HPGL</string>
   </property>
  </action>
  <action name="actionImport_HPGL">
   <property name="text">
    <string>Import HPGL</string>
   </property>
  </action>
  <action name="action_penConfig">
   <property name="text">
    <string>Pen Config</string>
   </property>
  </action>
  <action name="action_plotterConfig">
   <property name="text">
    <string>Plotter Config</string>
   </property>
  </action>
  <action name="action_viewConfig">
   <property name="text">
    <string>View Config</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
